src/index.html

```diff
@@ -14,5 +14,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--sunday calendar--31">
```

Comment:
Modifier should be a bit different in the task
![image](https://github.com/user-attachments/assets/6bc3a21e-5fc9-47cc-83fb-264fcc8e0c12)